src/wrapper/descriptors/class_desc.rs

```diff
@@ -23,6 +23,7 @@ impl<'a, 'b> Desc<'a, JClass<'a>> for JObject<'b> {
     }
 }
 
+/// This conversion assumes that the `GlobalRef` is a pointer to a class object.
```

Comment:
I'd replace «a pointer to» with «a reference to» since Java doesn't have pointers.